### PR TITLE
Localize the onboarding wizard

### PR DESCRIPTION
**Pull request by atimmer**
This makes it possible to translate the next and previous buttons

Inspired by i18n-calypso, but https://github.com/Automattic/i18n-calypso/issues/19 blocks us from using it.

Required for https://github.com/Yoast/wordpress-seo/pull/5582

### CODE REVIEW

File: utils/i18n.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	}
+}
+
+/**
+ * A higher order component to add a translate function.
+ *
+ * @param {Object} ComposedComponent The original React component.
+ */
+export function localize( ComposedComponent ) {
+	let componentName = ComposedComponent.displayName || ComposedComponent.name || '';
```
Comment (jcomack) [inline utils/i18n.js]: Please use double quotes.

File: utils/i18n.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	let i18nProps = {
+		translate: translate,
+	};
+
+	return React.createClass( {
+		displayName: 'Localized' + componentName,
```
Comment (jcomack) [inline utils/i18n.js]: See previous comment.

**Comment by jcomack** [comment]
CR done. 👷

Acceptance done as well. Translations work, however I do get the following error.

yoast_seo_ _setup_wizard

I cannot get rid of this issue despite clearing out my `node_modules` and reinstalling. Also, for some reason, I cannot see why the Travis builds keep failing, as Travis says the repository doesn't exist.

**Comment by atimmer** [comment]
The error has nothing to do with this PR. I have fixed it on `develop` when not using `npm link`.

**Pull request merged**